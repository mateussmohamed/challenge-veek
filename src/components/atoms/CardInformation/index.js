import styled from 'styled-components'

const CardInformation = styled.div`
  margin-left: 15px;
  display: flex;
  align-items: flex-start;
`

export default CardInformation
