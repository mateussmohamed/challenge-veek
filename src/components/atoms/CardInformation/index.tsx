import styled from 'styled-components'

export const CardInformation = styled.div`
  margin-left: 15px;
  display: flex;
  align-items: flex-start;
`
