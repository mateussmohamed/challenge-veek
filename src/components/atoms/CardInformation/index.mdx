---
name: CardInformation
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardInformation from './index'

# CardInformation
<PropsTable of={CardInformation} />

<Playground>
  <CardInformation />
</Playground>
