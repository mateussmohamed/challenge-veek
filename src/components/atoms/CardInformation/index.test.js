import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardInformation from './index'

describe('[<CardInformation />]', () => {
  const wrap = (props = {}) => shallow(<CardInformation {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
