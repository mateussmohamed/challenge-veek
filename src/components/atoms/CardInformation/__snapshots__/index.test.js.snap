// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardInformation />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardInformation-sc-1c34v2c-0",
        "isStatic": true,
        "rules": Array [
          "margin-left:15px;display:flex;align-items:flex-start;",
        ],
      },
      "displayName": "CardInformation",
      "render": [Function],
      "styledComponentId": "CardInformation-sc-1c34v2c-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
